ydb/core/mind/dynamic_nameserver.cpp

```diff
@@ -19,43 +19,38 @@ static void ResetInterconnectProxyConfig(ui32 nodeId, const TActorContext &ctx)
 
 void TDynamicNodeResolverBase::Bootstrap(const TActorContext &ctx)
 {
-    auto dinfo = AppData(ctx)->DomainsInfo;
-
-    NTabletPipe::TClientRetryPolicy retryPolicy = {
-        .RetryLimitCount = 12,
-        .MinRetryTime = TDuration::MilliSeconds(50),
-        .MaxRetryTime = TDuration::Seconds(2)
-    };
+    LOG_D("New cache miss: nodeId=" << NodeId << ", deadline=" << Deadline);
```

Comment:
Обычно мы пишем так:
```suggestion
    LOG_D("New cache miss"
        << ": nodeId# " << NodeId
        << ", deadline# " << Deadline);
```